corehq/apps/domain/auth.py

```diff
@@ -19,53 +19,48 @@
 TOKEN = 'token'
 
 
-def determine_authtype_from_header(request, default=DIGEST):
+def determine_authtype_from_header(request, support_non_j2me_digest=True):
```

Comment:
Might be me, but I find this renaming of the parameter to be more confusing than not. Perhaps we could create two additional functions that call a private function with the right params but leave this as-is?

Reply:
as per my comment below i think it is clearer to call this default